pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,99 @@
+import NextAuth from "next-auth"
+import { JWT } from "next-auth/jwt"
+import ADProvider from "next-auth/providers/azure-ad"
+import qs from "query-string"
+
+import { config } from "../../../config"
```

Comment:
Base url, I miss you 😢

Reply:
feel free to open a pr to set up the config 😅

Reply:
We should try it someday, maybe it Just Works(™)